### PR TITLE
October 4 Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 7c519da |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-536.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/amplify_relnotes/20241004_amplify.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+* Go to [https://repository.axway.com/catalog?q=agents](https://repository.axway.com/catalog?q=agents).
+
+### Axway Agent SDK
+
+The Agent SDK was updated to address security fixes in this release. The following enhancements and bug fixes are available in this release.
+
+#### Enhancements for Agent SDK
+
+* **Identity Provider Status** - A new Identity Provider status has been added to indicate a Pending, Success, or Error status.  For example, an error status will be set if an issue is found with the Identity Provider provider configuration of the URL or IdP Secret.
```
Comment (jcollins-axway) [inline content/en/docs/amplify_relnotes/20241004_amplify.md:59]: This is more of a SaaS update and less of an SDK update

File: content/en/docs/amplify_relnotes/20241004_amplify.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+
+### Agent versions
+
+Use command `agentName --version` to display version information in the agents.
+
+| Environment type                        | Latest on-premise agent version <br />(based on Amplify Agents SDK v1.1.101 <br />unless otherwise noted) | Minimum supported agent version <br />(based on Amplify Agents SDK v1.1.83) |
+|--------------------------------------------|--------------------------|------------------|
+| Axway API Management 7.6.2 SPx and 7.7 SPx | DA=v1.2.6 / TA=v1.2.5    | DA=v1.1.80 / TA=v1.1.81          |
+| AWS Gateway using SDK 2.0                  | DA=v1.2.5 / TA-v1.2.5    | DA=v1.1.64 / TA-v1.1.67          |
+| Azure latest release                       | DA=v1.3.1 / TA=v1.3.1    | DA=v1.1.70 / TA=v1.1.76          |
```
Comment (jcollins-axway) [inline content/en/docs/amplify_relnotes/20241004_amplify.md:21]: Need to rerelease Azure once Dale figures out pipeline issues and a new release is made.

File: content/en/docs/amplify_relnotes/20241004_amplify.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+| Istio 1.9.5                                | DA=v1.1.4 / TA=v2.1.4    | DA=v1.0.36 / TA=v2.0.82          |
+| Apigee Edge                                | 1.0.1                    | 0.1.38                           |
+| Mulesoft Anypoint platform v3              | 1.2.4                    | 1.1.48                           |
+| Software AG webMethods                     | 1.0.3                    | 1.0.0                            |
+| Kong Gateway                               | 1.1.4                    | 1.0.9                            |
+| GitLab                                     | 1.1.4                    | 1.0.6                            |
+| Kafka Cluster                              | DA=v1.1.4 / TA=v1.1.4    | DA=v1.0.6 / TA=v1.0.6            |
+| IBM API Connect Gateway                    | DA=v1.1.4 / TA=v1.1.4    | DA=v1.0.0 / TA=v1.0.0            |
```
Comment (jcollins-axway) [inline content/en/docs/amplify_relnotes/20241004_amplify.md:29]: IBM, Kafka, and soon Azure DA/TA will always have the same versions, so you could reduce their current versions to the one.

File: content/en/docs/amplify_relnotes/20241004_amplify.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+|-------------|--------------|---------------------------------------------------|
+|             | APIGOV-28892 | **Issue**: In cases where a dash was used in the filter name, the agent would not filter the API name properly. <br />**Resolution**: A fix was made to enable the matching with a dash in the filter name.  |
+
+#### Known issues for Agent SDK
+
+There are no known limitations in this release.
+
+### On-premise agents
+
+All on-premise agents have been updated with the Agent SDK version 1.1.99.
```
Comment (jcollins-axway) [inline content/en/docs/amplify_relnotes/20241004_amplify.md:73]: 1.1.101 now

File: content/en/docs/amplify_relnotes/_index.md [hunk 5]
```diff
@@ -20,6 +20,78 @@ To view the *Administration Release Notes* [click here](https://docs.axway.com/b
...
+| AWS Gateway using SDK 2.0                  | DA=v1.2.3 / TA-v1.2.3    | DA=v1.1.64 / TA-v1.1.67          |
+| Azure latest release                       | DA=v1.2.3 / TA=v1.2.3    | DA=v1.1.70 / TA=v1.1.76          |
+| Istio 1.9.5                                | DA=v1.1.3 / TA=v2.1.3    | DA=v1.0.36 / TA=v2.0.82          |
+| Apigee Edge                                | 1.0.0                    | 0.1.38                           |
+| Mulesoft Anypoint platform v3              | 1.2.3                    | 1.1.48                           |
+| Software AG webMethods                     | 1.0.2                    | 1.0.0                            |
+| Kong Gateway                               | 1.1.3                    | 1.0.9                            |
+| GitLab                                     | 1.1.3                    | 1.0.6                            |
+| Kafka Cluster                              | DA=v1.1.3 / TA=v1.1.3    | DA=v1.0.6 / TA=v1.0.6            |
+| IBM API Connect Gateway                    | DA=v1.1.3 / TA=v1.1.3    | DA=v1.0.0 / TA=v1.0.0            |
```
Comment (jcollins-axway) [inline content/en/docs/amplify_relnotes/_index.md:41]: some of these versions are not in sync with the latest release note

File: content/en/docs/amplify_relnotes/20241004_amplify.md (hunk 1 above)
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/20241004_amplify.md]: ok I moved it to the SaaS section

File: content/en/docs/amplify_relnotes/20241004_amplify.md (hunk 3 above)
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/20241004_amplify.md:29]: done

File: content/en/docs/amplify_relnotes/_index.md (hunk 5 above)
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/_index.md:41]: Lisa updates the index file to match after our edits

**Review by georgiana-caras — APPROVED** [review]

**Pull request merged**